composer.json

```diff
@@ -20,7 +20,7 @@
         "hhvm/hacktest": "^1.3"
     },
     "require": {
-        "hhvm": "^4.25",
+        "hhvm": "^4.29",
```

Comment:
Not strictly necessary, but I really want people with HHVM <4.29 to get the linters. I don't know if there's a better way to ensure that.

We can revert this to "^4.25" in a future release if

- anything useful (new linters/migrations) is added to HHAST so there's any value in doing so
- enough time has passed so that we're no longer concerned about people still using references